packages/dds/register-collection/src/consensusRegisterCollectionFactory.ts

```diff
@@ -58,3 +62,16 @@ export class ConsensusRegisterCollectionFactory implements IConsensusRegisterCol
 		return collection;
 	}
 }
+
+/**
+ * {@inheritdoc IConsensusRegisterCollection}
```

Comment:
```suggestion
 * {@inheritDoc IConsensusRegisterCollection}
```

Reply:
We seem to use both, but inheritDoc is more common. CHanged.